### PR TITLE
Improve docs

### PR DESCRIPTION
**Pull request by rylev**
Screenshot 2020-03-27 at 13 49 56

### CODE REVIEW

File: crates/kubelet/src/provider.rs [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+    }
+
+    /// Resolve the environment variables for a container.
+    ///
+    /// This generally should not be overwritten unless you need to handle
+    /// environment variable resolution in a special way, such as allowing
+    /// custom Downward API fields.
+    ///
+    /// It is safe to call from within your own providers.
+    // TODO: Finish secrets, configmaps, and resource fields
```
Comment (thomastaylor312) [inline crates/kubelet/src/provider.rs]: I think this TODO is no longer needed

File: crates/kubelet/src/status.rs [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+use k8s_openapi::api::core::v1::{
+    ContainerState, ContainerStateRunning, ContainerStateTerminated, ContainerStateWaiting,
+};
+use k8s_openapi::apimachinery::pkg::apis::meta::v1::Time;
+
+use std::collections::HashMap;
+
+/// Describe the status of a workload.
+///
+/// Phase captures the lifecycle aspect of the workload, while
```
Comment (thomastaylor312) [inline crates/kubelet/src/status.rs]: This comment is no longer valid as we don't capture phase directly. It should be updated to reflect that container statuses is part of it

File: crates/oci-distribution/src/client.rs [hunk 3]
```diff
@@ -0,0 +1,394 @@
+//! OCI distribution client
```
Comment (thomastaylor312) [inline crates/oci-distribution/src/client.rs]: We should probably add a note that this is not a full implementation yet, but we want it to be

File: crates/oci-distribution/src/manifest.rs [hunk 4]
```diff
@@ -1,3 +1,4 @@
+//! Oci Manifest
```
Comment (thomastaylor312) [inline crates/oci-distribution/src/manifest.rs]: Nit: Capitalize OCI

File: crates/kubelet/src/module_store.rs [hunk 5]
```diff
@@ -8,13 +11,47 @@ use std::convert::TryFrom;
 use std::path::{Path, PathBuf};
 use std::sync::Arc;
 
+/// A store of container modules.
+///
+/// This provides the ability to get a module's bytes given an image `Reference`.
```
[the remark this answers is not in the document]
Comment (rylev) [reply crates/kubelet/src/module_store.rs]: Yep, will do

**Pull request merged**